laba 5.txt

```diff
@@ -0,0 +1,81 @@
+#include <iostream>
+
+class Musician {
+
+private:
+    std::string name;
+    int price;
+
+public:
+    Musician(std::string name1,int price1){
+        name=name1;
+        price=price1;
+    }
+    
+    std::string get_name() {return name;};
```

Comment:
print return name in a new line  like this
std::string get_name() {
return name;
}